src/madpack/madpack.py

```diff
@@ -995,19 +996,20 @@ def run_unit_tests(args, testcase):
         Run unit tests.
     """
     if not _is_madlib_installation_valid_for_tests(args['schema'],
-                                                   args['db_madlib_ver']):
+                                                   args['db_madlib_ver'],
+                                                   'unit-tests'):
         return
     info_(this, "> Running unit-test scripts for:", verbose)
     modset = _get_modset_for_tests(testcase, 'test_')
     # Loop through all modules and run unit tests
-    _process_py_sql_files_in_modules(modset, {'madpack_cmd':'unit-test'})
+    _process_py_sql_files_in_modules(modset, {'madpack_cmd':'Unit-test'})
```

Comment:
Changing `unit-test` to `Unit-test` causes a failure when we try to run madpack with `unit-test` option. It's probably because the check for madpack command in function `_process_py_sql_files_in_modules` is case sensitive.